### PR TITLE
Support arb and eth

### PR DESCRIPTION
**Pull request by jihoonsong**

### CODE REVIEW

File: contracts/index/bases/IndexStrategyUpgradeable.sol [hunk 1]
```diff
@@ -59,7 +60,9 @@ abstract contract IndexStrategyUpgradeable is
     IIndexOracle public oracle;
     uint256 public equityValuationLimit;
 
-    uint256[8] private __gap;
+    address public constant NATIVE = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
```
[the remark this answers is not in the document]
Comment (jihoonsong) [reply contracts/index/bases/IndexStrategyUpgradeable.sol:63]: We can also accept wNATIVE but ,for now, we'll receive ETH, ARB, and USDC.

File: scripts/constants/deposit-tokens.ts [hunk 2]
```diff
@@ -2,8 +2,28 @@ import ArbitrumTokens from "../../constants/arbitrum/addresses/Tokens.json"
...
+    new Map<string, string>([
+      ["0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E", "1"], // USDC on Avalanche
+      ["0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7", "0.1"], // wAVAX on Avalanche
+    ]),
+  ],
+  [
+    "arbitrum-one",
+    new Map<string, string>([
+      ["0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8", "1"], // USDC on Arbitrum
+      ["0x82aF49447D8a07e3bd95BD0d56f35241523fBab1", "0.0006"], // wETH on Arbitrum
```
[the remark this answers is not in the document]
Comment (jihoonsong) [reply scripts/constants/deposit-tokens.ts:25]: It's for both testing and real. It doesn't have to be exact $1, it's okay as long as it is few dollars. We can change the amount whenever we want :)

**Comment by jihoonsong** [comment]
I tested if all upgrades will work as expected and it's good to go.

**Comment by ferencdg** [comment]
What is the relationship of this PR and the #70? I see **some** of the commits were selectively cherry-picked into this PR. I don't mind having only 1 PR with 2 features included, but right now I don't even know if I merge #70 first, and then this one, it will succeed or not.

**Comment by jihoonsong** [comment]
I needed to make indices to support multiple tokens during your OOO. The cherry picks are because of dependencies of different projects, which is a tech debt I pointed out long ago. This PR was supposed to be merged before I left but somehow it couldn't.

**Comment by ferencdg** [comment]
> I needed to make indices to support multiple tokens during your OOO. The cherry picks are because of dependencies of different projects, which is a tech debt I pointed out long ago. This PR was supposed to be merged before I left but somehow it couldn't.

so how about: first the #70 PR is merged in (after the comments are addressed), and then you could rebase this, and we could merge it

**Comment by jihoonsong** [comment]
This one is ready and supposed to be merged before #70. I suggest to merge this one first and you can add change on #70 as much as you want including rebasing if necessary, and then merge it.

**Comment by ferencdg** [comment]
> This one is ready and supposed to be merged before #70. I suggest to merge this one first and you can add change on #70 as much as you want including rebasing if necessary, and then merge it.

Those 2 are you PRs Jihoon, I should not be the one rebasing it. If you want we can merge this one first, but then the other PR needs to be fixed.

**Comment by ferencdg** [comment]
I can give you a counter example when **manual** rebasing is needed:
1. main has function f()
2. in PR1 you add a parameter to 'a' to f(a)
3. You cherry pick this change from PR1 to PR2, and add later a commit to add parameter 'b' to f('a','b') in PR2
4. You merge PR2 to main
5. You try to merge PR1 to main: what happens now? f('a') is brought back to main without any merging error(but with probably compile errors)? Manual rebase is needed?

It is not an ideal way to cherry-pick certain commits from other PRs.

**Comment by jihoonsong** [comment]
Sounds like lecturing. Very not appreciated and very not good. Also, it's okay to do something when you know what you're doing.

#70 is part of you work Daniel. I'm not the one who has to merge it. You can adopt it, or rebase it, or abandon it, or make change as much as you want.

**Comment by ferencdg** [comment]
for the record there were many merge conflicts when rebasing to main

File: contracts/index/bases/IndexStrategyUpgradeable.sol (hunk 1 above)
Comment (ferencdg) [inline contracts/index/bases/IndexStrategyUpgradeable.sol:63]: this is a constant, not a storage variable, no need to adjust gap

**Pull request merged**